doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -81,6 +85,48 @@ Tactics
       variant is very useful for getting a better understanding of automation, or
       to know what lemmas/assumptions were used.
 
+.. _info_auto_not_exact:
+
+      The tactics shown in the info or debug output currently don't
+      correspond exactly to the variants that proof search tactics such
+      as :tacn:`auto` use, but they are close.
+
+      Occasionally the tactics shown (after removing tactics that were
+      backtracked) may not always work as a replacement for the proof search
+      tactic.  For example:
+
+      .. example:: `info_auto` output that isn't a valid proof
+
+         The output isn't accepted as a proof because the conversion
+         constraints are solved by default after every statement but
+         are not solved internally by :tacn:`auto` as it searches for
+         a proof.
+
+         .. rocqtop:: in
+
+            Create HintDb db.
+
+            Hint Resolve conj : db.
+            Hint Resolve eq_refl : db.
+
+            Goal forall n, n=1 -> exists x y : nat, x = y /\ x = 0.
+            intros.
+            do 2 eexists; subst.      (* Fix 2: replace with "eexists; subst." twice *)
```

Comment:
```suggestion
            do 2 (eexists; subst).      (* Fix 2: replace with "eexists; subst." twice *)
```

Is this the intended parse? Even if the rules are clear, I think it would be helpful to make it explicit here.